docs/user-guide/concepts/agents/structured-output.md

```diff
@@ -74,79 +54,159 @@ class PersonInfo(BaseModel):
+```
+
+
+###  Structured Output agent with other tools
+
+Combine structured output with tool usage to format tool execution results:
+
+```python
+from strands import Agent
+from strands_tools import calculator
+from pydantic import BaseModel, Field
+
+class MathResult(BaseModel):
+    operation: str = Field(description="the performed operation")
+    result: int = Field(description="the result of the operation")
+
+tool_agent = Agent(
+    tools=[calculator]
 )
+res = tool_agent("What is 42 + 8", structured_output_model=MathResult)
 ```
 
-For a complete list of supported content types, please refer to the [API Reference](../../../api-reference/types.md#strands.types.content.ContentBlock).
+###  Many Invocations with a Single Agent and different Structured Output types
+
+Reuse a single agent instance with different structured output models for varied extraction tasks:
+
+```python
+from strands import Agent
+from pydantic import BaseModel, Field
+
+class Person(BaseModel):
+    """A person's basic information"""
+    name: str = Field(description="Full name")
+    age: int = Field(description="Age in years", ge=0, le=150)
+    email: str = Field(description="Email address")
+    phone: Optional[str] = Field(description="Phone number", default=None)
+
+class Task(BaseModel):
+    """A task or todo item"""
+    title: str = Field(description="Task title")
+    description: str = Field(description="Detailed description")
+    priority: str = Field(description="Priority level: low, medium, high")
+    completed: bool = Field(description="Whether task is completed", default=False)
+
+
+agent = Agent()
+person_res = agent("Extract person: John Doe, 35, john@test.com", structured_output_model=Person)
+task_res = agent("Create task: Review code, high priority, completed", structured_output_model=Task)
+```
 
 
 ### Using Conversation History
 
-Structured output can work with existing conversation context:
+Extract structured information from prior conversation context without repeating questions:
 
 ```python
+from strands import Agent
+from pydantic import BaseModel
+from typing import Optional
+
 agent = Agent()
 
 # Build up conversation context
 agent("What do you know about Paris, France?")
 agent("Tell me about the weather there in spring.")
 
-# Extract structured information with a prompt
 class CityInfo(BaseModel):
     city: str
     country: str
     population: Optional[int] = None
     climate: str
 
-# Uses existing conversation context with a prompt
-result = agent.structured_output(CityInfo, "Extract structured information about Paris")
+# Extract structured information from the conversation
+result = agent(
+    "Extract structured information about Paris from our conversation",
+    structured_output_model=CityInfo
+)
+
+print(f"City: {result.structured_output.city}")     # "Paris"
+print(f"Country: {result.structured_output.country}") # "France"
 ```
 
-### Complex Nested Models
+### Nested Models
```

Comment:
I'd argue to remove this or keep this as one of the examples before; having this as a separate example is just bloat IMHO